service_info/templates/aldryn_search/includes/search_items.html

```diff
@@ -0,0 +1,34 @@
+{% load i18n highlight %}
+
+{% for result in page_obj.object_list %}
+	<div class="card-pad blog-card">
+		<div class="card">
```

Comment:
nitpick: The indentation in this file could probably be reduced. The file above is a mix of 2 or 4, either of which I'd prefer over 8 :)